### ISSUE TITLE
[1.12.1] roots-2-0.100 crash on startup MP server

### ISSUE THREAD

**Issue by CplPibald** [RootsTeam/Roots#250]
Just downloaded build 100 and it is crashing on MP server startup.

```
Description: Exception in server tick loop

net.minecraftforge.fml.common.LoaderExceptionModCrash: Caught exception from Roots 2 (roots)
Caused by: java.lang.NoClassDefFoundError: net/minecraft/client/entity/EntityPlayerSP
        at java.lang.Class.getDeclaredConstructors0(Native Method)
        at java.lang.Class.privateGetDeclaredConstructors(Class.java:2671)
        at java.lang.Class.getConstructor0(Class.java:3075)
        at java.lang.Class.getConstructor(Class.java:1825)
        at net.minecraftforge.fml.common.registry.EntityEntry.init(EntityEntry.java:49)
        at net.minecraftforge.fml.common.registry.EntityEntry.<init>(EntityEntry.java:41)
        at net.minecraftforge.fml.common.registry.EntityRegistry.doModEntityRegistration(EntityRegistry.java:183)
        at net.minecraftforge.fml.common.registry.EntityRegistry.registerModEntity(EntityRegistry.java:151)
        at teamroots.roots.RegistryManager.registerAll(RegistryManager.java:148)
        at teamroots.roots.proxy.CommonProxy.preInit(CommonProxy.java:26)
        at teamroots.roots.proxy.ServerProxy.preInit(ServerProxy.java:10)
        at teamroots.roots.Roots.preInit(Roots.java:55)
```

**Comment by oitsjustjose** [comment]
I get this as well, unfortunately. Let me know if I can help!

**Comment by twoworlds** [comment]
Same here on my 1.12.1 Server.

Crash Log : https://pastebin.com/hQsnT0R2

**Comment by elucent** [comment]
I'm stupid. Give me a bit.

**Comment by elucent** [comment]
Fixed for Roots 2 0.101. Will be on CF shortly.
